### ISSUE TITLE
[Feature Request] Add Support for Minecraft 1.8.8

### ISSUE THREAD

**Issue by Ant4rez** [MagmaGuy/EliteMobs#28]
Hey there!,

I just want to request if it possible to add support for minecraft 1.8.8. I would really like to use elitemobs on my factions servers, i love this plugin this plugin that i already used before on a server with some friends, it's gives a great hardcore survival experience, but now i wanted to use it on a tacospigot server, thank you so much:D.

Greetings.

**Comment by MagmaGuy** [comment]
Unfortunately it is quite literally impossible to add support for 1.8 without rewriting all of EliteMobs as it is 100% reliant on the new combat system, and uses API features that only became available in 1.14+.

As such, adding support for 1.8 basically requires a full rewrite. with independent maintenance and updating then further required to maintain parity with new features as they come along.

In other words, not going to happen. I recommend updating to a version that isn't older than some Minecraft players.
